src/main/java/org/example/DBController.java

```diff
@@ -128,6 +129,142 @@ public boolean updateCustomerFunds(int customerId, int newAmount) {  //void
         return false;
     }
 
+    public void createTransactionAudit(int senderId, String senderName, int moneyInCents, int getterId, String getterName) throws SQLException {
+        String sql = "INSERT INTO transactions_audit (sender_id, sender_nickname, amount_sent, receiver_id, receiver_nickname) " + "VALUES (?, ?, ?, ?, ?)";

```

Comment:
You don't need to concatenate these strings:
```suggestion
        String sql = "INSERT INTO transactions_audit (sender_id, sender_nickname, amount_sent, receiver_id, receiver_nickname) VALUES (?, ?, ?, ?, ?)";
```
Or you can play around with string concatenation (see https://www.baeldung.com/java-multiline-string) to divide this string into separate sections. But that's generally not necessary for something of this scope.